package.json

```diff
@@ -142,7 +142,7 @@
     "@babel/cli": "7.0.0-beta.38",
     "@babel/core": "7.0.0-beta.38",
     "@babel/node": "^7.0.0-beta.38",
-    "@babel/preset-env": "^7.0.0-beta.38",
+    "@babel/preset-env": "7.0.0-beta.38",
```

Comment:
@aldeed the error below gets thrown in CI test. I found that removing the `^` to ensure preset-env is exactly 7.0.0-beta.38 resolves the issue. Is this a good way to resolve this?
```
[BABEL] /home/circleci/reaction-app/imports/plugins/core/graphql/server/resolvers/account/Query/administrators.test.js:
Requires Babel "^7.0.0-beta.41", but was loaded with "7.0.0-beta.38". 
You'll need to update your @babel/core version. (While processing: "/home/circleci/reaction-app/node_modules/@babel/preset-env/lib/index.js")
```